hello-world/hello_world_test.rb

```diff
@@ -1,8 +1,12 @@
 require 'minitest/autorun'
 require_relative 'hello_world'
 class HelloWorldTest < Minitest::Test
+  def test_no_strange_name
+    assert_equal 'Hello, !', HelloWorld.hello(''), 'Greeting given an empty string as a name'
+  end
+
   def test_no_name
-    assert_equal 'Hello, world!', HelloWorld.hello(''), 'Greeting given no name'
+    assert_equal 'Hello, world!', HelloWorld.hello, 'Greeting given an empty string as a name'
```

Comment:
I think my prior messages (wherever I submitted messages in tests) were trying to be descriptive of the reason the test failed.).  Having independent tests are important, that gives the behavior wanted.  The message, feedback on why it failed.

So the action points on this then are to have tests for default name (or no name) and a purposefully given blank name.  But no name given should be covered due to the 'world' being the default name?  Looking at the code again... 

With one failure, with changes, it can read like this:
```
HelloWorldTest#test_no_name [hello_world_test.rb:9]:
When given no name, the default should happen.
Expected: "Hello, world!"
  Actual: "Hello, !"
```
I will do a push. :)